openassessment/assessment/api/staff.py

```diff
@@ -160,6 +198,62 @@ def get_assessment_scores_by_criteria(submission_uuid):
         raise StaffAssessmentInternalError(error_message)
 
 
+def get_submission_to_assess(course_id, item_id, scorer_id):
+    """Get a submission for staff evaluation.
+
+    Retrieves a submission for assessment for the given staff member. This will
+    not return a submission submitted by the requesting scorer.
+
+    Args:
+        submission_uuid (str): The submission UUID from the student
```

Comment:
Can staff be a student in the same class?  Or is all of this just peer leftovers?